docs/how_to/process_video.md

```diff
@@ -1 +1,74 @@
-🚧 Page under construction.
+With supervision, you can run inference on or otherwise process video frames with ease. In this guide, we will show how to run inference on each frame in a video and save the results to a new file.
+
+To process video, we will use the [sv.process_video](https://supervision.roboflow.com/utils/video/#process_video) method.
+
+## Define a Callback Function
+
+In this guide, we will use a YOLOv8 checkpoint model to identify objects in a video. To process this video, we need to define a function that accepts a video frame and runs inference.
+
+Create a new Python file and add the following code:
+
+```python
+import supervision as sv
+import numpy as np
+from ultralytics import YOLO
+
+model = YOLO("yolov8x.pt")
+
+def callback(scene: np.ndarray, index: int) -> np.ndarray:
+    results = model(scene)[0]
+    detections = sv.Detections.from_ultralytics(results)
+
+    bounding_box_annotator = sv.BoundingBoxAnnotator()
+    label_annotator = sv.LabelAnnotator()
+
+    labels = [
+        results.names[class_id]
+        for class_id
+        in detections.class_id
+    ]
```

Comment:
Let's let the annotator resolve the names itself. If a user plugs in a model without class_names, it will work just as well.

Here's an example from another file:

```
bounding_box_annotator = sv.BoundingBoxAnnotator()
label_annotator = sv.LabelAnnotator()

annotated_image = bounding_box_annotator.annotate(
    scene=image, detections=detections)
annotated_image = label_annotator.annotate(
    scene=annotated_image, detections=detections)
```

It auto-determines if there's a class name and displays it.